### DISCUSSION TITLE
Series cine view

### DISCUSSION THREAD

**Discussion by Ravindra-Desai128** [nroduit/Weasis#456] [General]
Hi @nroduit , Thank you so much for creating this kind of framework,  By any chance auto series cine view option is there in this viewer.
after completing one series,  it should go to next series in cine view,  as of now only list of instances in one series able to cine view right?

**Comment by nroduit** [comment]
No, [IHE Basic Image Review profile](http://www.ihe.net/uploadedFiles/Documents/Radiology/IHE_RAD_Suppl_BIR.pdf) says "play the frames as a continuously cycling loop, either forward or reverse" (in line 1030)

**Reply by Ravindra-Desai128** [reply]
@nroduit ok thanks;  is that possible to give button (cine go to next series) instead of (cine go to end) in Cine Bar. could you please answer for this.


Thanks in Advance.

**Comment by nroduit** [comment]
Browsing for patients/exams/series is something that's been on the to-do list for a long time. 
The resources on this project are very limited. We need contributors for this kind of new feature.

**Reply by nroduit** [reply]
See https://github.com/nroduit/Weasis/issues/483

**Comment by Ravindra-Desai128** [comment]
Thanks
